backend/core/views.py

```diff
@@ -1307,7 +1307,9 @@ class RiskScenarioViewSet(BaseModelViewSet):
     ordering_fields = ordering
 
     def _perform_write(self, serializer):
-        if not serializer.validated_data.get("ref_id"):
```

Comment:
When we create an applied control from a scenario, the validated data only contains the applied control created, not the risk assessmentq so an error occured